backend/routes/assignments.js

```diff
@@ -147,4 +147,29 @@ router.get('/grades/:assignmentId', auth, async (req, res) => {
 
 })
 
+/**
+ * Fetch all grades for given assignment
+ * Route: GET
+ * Params: assignmentId
+ */
+router.get('/grades/all/:assignmentId', auth, async (req, res) => {
```

Comment:
Not that this is necessarily dealbreaking, but specs say the route should just be:
/assignments/grade/:assignmentId
Is there a reason for the /all that I might just not be aware of?

Reply:
Yes, there is a conflicting route with the same pattern. Due to the same route pattern, the route in #13 will interpret the request first and the request never comes to the above method. Hence I had to change the path